README.md

```diff
@@ -96,80 +97,159 @@ This is a sample of how you'd use supabase-py. Functions and tests are WIP
 ## Authenticate
 
 ```python
-from supabase import create_client, Client
+import os
+from supabase import SupabaseClient
 
 url: str = os.environ.get("SUPABASE_TEST_URL")
 key: str = os.environ.get("SUPABASE_TEST_KEY")
-supabase: Client = create_client(url, key)
+
+supabase: SupabaseClient = SupabaseClient(url, key)
+
 # Create a random user login email and password.
-random_email: str = "3hf82fijf92@supamail.com"
-random_password: str = "fqj13bnf2hiu23h"
+random_email: str = "user@supamail.com"
```

Comment:
Hm is there any significant benefit that we gain from changing the email and password in the test examples?

Reply:
Since these are dummy values, readability is important. I personally think it's easier to read "user@supabase.com" over the random one.